extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -103,6 +106,9 @@ export class ProjectsController {
 	public async buildProject(context: Project | dataworkspace.WorkspaceTreeItem): Promise<string> {
 		const project: Project = this.getProjectFromContext(context);
 
+		const date = new Date();
+		TelemetryReporter.createActionEvent(TelemetryViews.ProjectController, 'buildStarted').send();
```

Comment:
That's true.  Just less processing on the server side since we can trivially isolate duration here.  It's cheap enough to capture and makes writing useful queries easier.  Think it's worth doing that calculation in Kusto?

Reply:
In general we should optimize for sending less events - every event we send contains a bunch of extra metadata that just takes up unnecessary space and adds extra network overhead. Processing on the server side is relatively cheap in comparison.